File: Backend/src/main/java/com/gameshub/controller/DTO/ProductDTO.java
package com.gameshub.controller.DTO;

import lombok.Data;
import java.time.LocalDate;
import java.util.List;

@Data
public class ProductDTO {
    private int id;
    private float price;
    private String description;
    private String title;
    private int count;
    private int sellerID;
    private LocalDate postDate;
    private String category;
    private List<String> urls;
}

File: Frontend/src/Controller/DTO/ProductDTO/ProductDTO.ts
export interface ProductDTO {
  id: number;
  dateReceived: string; // Change to match LocalDate format in Java (e.g., 'yyyy-MM-dd')
  status: string;
  requestType: string;
  title: string;
  price: number;
  description: string;
  postDate: string; // Change to match LocalDate format in Java (e.g., 'yyyy-MM-dd')
  count: number;
  sellerId: number;
  category: string;
  urls: string[];
  code: string;
}